packages/ui/components/avatar/AvatarGroup.tsx

```diff
@@ -11,19 +11,21 @@ export type AvatarGroupProps = {
   }[];
   className?: string;
   accepted?: boolean;
+  truncateAfter?: number;
 };
 
 export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
   const avatars = props.items.slice(0, 4);
   const LENGTH = props.items.length;
+  const truncateAfter = props.truncateAfter || 4;
```

Comment:
Make this more reusable - we often use 4 but makes this available if we ever need more or less